packages/extension/src/companion/companionRequest.tsx

```diff
@@ -1,22 +1,23 @@
-import request from 'graphql-request';
 import browser from 'webextension-polyfill';
 import { initialDataKey } from '@dailydotdev/shared/src/lib/constants';
 import { ExtensionMessageType } from '@dailydotdev/shared/src/lib/extension';
+import { gqlRequest } from '@dailydotdev/shared/src/graphql/common';
+import { graphqlUrl } from '@dailydotdev/shared/src/lib/config';
 
 const proxyRequest = {
   apply(_, __, args) {
-    const { [initialDataKey]: initial, ...variables } = args?.[2];
+    const { [initialDataKey]: initial, ...variables } = args?.[1];
 
     browser.runtime.sendMessage({
       type: ExtensionMessageType.GraphQLRequest,
-      url: args?.[0],
-      document: args?.[1],
+      url: graphqlUrl,
+      document: args?.[0],
       variables,
-      headers: args?.[3],
+      headers: args?.[2],
     });
 
     return initial ?? null;
   },
 };
 
-export const companionRequest = new Proxy(request, proxyRequest);
+export const companionRequest = new Proxy(gqlRequest, proxyRequest);
```

Comment:
This one we'll have to double verify.
I think I tried this before and it might works, but best to double test it as the weird proxy was needed for some reason.

Reply:
I tried it yes, basically first argument (url) was always graphqlUrl which now is no longer needed since it is set on the client init.

Reply:
This proxy is so that all requests go through background script.